src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2274,9 +2275,14 @@ private void DynamoViewModelRequestShowPackageManager(object s, EventArgs e)
             {
                 packageManagerWindow.Navigate((e as OpenPackageManagerEventArgs).Tab);
             }
-            _pkgSearchVM.RefreshAndSearchAsync();
+            if (!_initialSearch)
```

Comment:
Only perform RefreshAndSearchAsync once. Since the VM is alive, we don't have to rerun this method.

Reply:
So RefreshAndSearchAsync will be called only once for every dynamo session? Users will have to restart dynamo to get the latest package changes which is not ideal. Any reason for doing this?

Reply:
Yes, indeed. I will bring that up during our catch-up. 

I can remove it from this PR, but I will propose to have a separate refresh button to handle that functionality.